src/lib/FlightTasks/tasks/FlightTask/FlightTask.hpp

```diff
@@ -166,7 +166,7 @@ class FlightTask : public ModuleParams
 	/**
 	 * Call this whenever a parameter update notification is received (parameter_update uORB message)
 	 */
-	void handleParameterUpdate()
+	void virtual handleParameterUpdate()
```

Comment:
I'm guessing you have a closed source task that needs to override to do some parameter checks when updating them.

I hope it's ok to stick to the convention of putting `virtual` at the beginning.